gui/src/rime-utils/FcitxRimeConfig.c

```diff
@@ -0,0 +1,272 @@
+#include "FcitxRimeConfig.h"
+#include "assert.h"
+#include "fcitx-config/xdg.h"
+#include "fcitx-utils/utils.h"
+#include <stdio.h>
+
+#ifdef __cplusplus
```

Comment:
you can use c++ in this file, I guess?

Reply:
Can I? The rime api is C API, so I have to call using C interface.